pom.xml

```diff
@@ -25,7 +25,7 @@
     <useBeta>true</useBeta>
 
     <!-- Jenkins Plug-in Dependencies Versions -->
-    <checks-api.version>1.5.0</checks-api.version>
+    <checks-api.version>1.6.0-rc209.3e34bea21942</checks-api.version> <!-- TODO https://github.com/jenkinsci/checks-api-plugin/pull/93 -->
```

Comment:
I'll release it once I figure out the build problem: https://github.com/jenkinsci/checks-api-plugin/pull/93#issuecomment-791898907